src/Handler.ts

```diff
@@ -84,14 +82,20 @@ export class Handler {
     // make the tooltip visible
     this.el.classList.add('visible', `${this.options.theme}-theme`);
 
-    const {x, y} = calculatePosition(
-      event,
-      this.el.getBoundingClientRect(),
-      this.options.offsetX,
-      this.options.offsetY,
-    );
+    let position: {x: number; y: number};
+
+    if (this.options.anchor === 'mark') {
+      position = calculatePositionRelativeToMark(handler, event, item, this.el.getBoundingClientRect(), this.options);
+    } else {
+      position = calculatePositionRelativeToCursor(
+        event,
+        this.el.getBoundingClientRect(),
+        this.options.offsetX,
+        this.options.offsetY,
+      );
+    }
```

Comment:
could this become 

```
const position = (this.options.anchor === 'mark' ? calculatePositionRelativeToMark : calculatePositionRelativeToCursor)(event, this.el.getBoundingClientRect(), this.options)
```

Reply:
Good catch

Reply:
The two methods can't have the exact same arguments since `calculatePositionRelativeToMark` needs the `handler` and the `item`.

Unless we want to pass the `handler` and `item` into `calculatePositionRelativeToCursor` but never use them.